eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -136,3 +143,23 @@ jobs:
             summaryFileLocation: '$(Build.SourcesDirectory)/**/Cobertura.xml'
           displayName: Publish Code Coverage to DevOps
           condition: and(succeededOrFailed(), eq(variables['CODE_COVERAGE'], 'enabled'))
+
+        - task: mspremier.BuildQualityChecks.QualityChecks-task.BuildQualityChecks@8
+          displayName: Check line coverage
+          inputs:
+            checkCoverage: true
+            coverageFailOption: fixed
+            coverageType: line
+            # Minimum baseline for line coverage
+            coverageThreshold: ${{ parameters.LineCoverageTarget }}
+          condition: and(succeededOrFailed(), eq(variables['CODE_COVERAGE'], 'enabled'), eq(variables['Skip.CoverageEnforcement'], ''))
```

Comment:
Can we have one Skip.LineCoverage and another Skip.BranchCoverage?   That would give us more flexibility